src/connectors/hits/connectHits.js

```diff
@@ -9,60 +9,43 @@ var customHits = connectHits(function render(params, isFirstRendering) {
   //   widgetParams,
   // }
 });
-search.addWidget(
-  customHits({
-    [ hitsPerPage = 20 ]
-  })
-);
+search.addWidget(customHits());
 Full documentation available at https://community.algolia.com/instantsearch.js/connectors/connectHits.html
 `;
 
 /**
- * @typedef {Object} CustomHitsWidgetOptions
- * @param {number} [hitsPerPage = 20] The number of hits to display per page
- */
-
-/**
  * @typedef HitsRenderingOptions
  * @property {Object[]} hits the hits contained in the last results from Algolia
  * @property {Object} results the complete results from Algolia
  * @property {InstantSearch} instantSearchInstance the instance of instantsearch on which the widget is attached
  * @property {Object} widgetParams all original options forwarded to rendering
  */
 
- /**
-  * Connects a rendering function with the hits business logic.
-  * @param {function(HitsRenderingOptions, boolean)} renderFn function that renders the hits widget
-  * @return {function(CustomHitsWidgetOptions)} a widget factory for hits widget
-  */
+/**
+ * Connects a rendering function with the hits business logic.
+ * @param {function(HitsRenderingOptions, boolean)} renderFn function that renders the hits widget
+ * @return {function} a widget factory for hits widget
```

Comment:
`A function that returns a widget` is what we decided. However it could be done now or during the documentation review (but that would be too bad to miss it)